[minor_change] Added Playbook Optimizing documentation (#641)
CiscoDevNet/ansible-aci: README.md

    delegate_to: localhost
```

> ## Optimizing Playbooks

Reviewer: Is optimizing the right wording? Because the "optimization" is mostly execution time optimization which also comes at a cost. Should we rephrase to something in the trend of `decrease playbook execution times`

Author: Yeah, optimization can have multiple elements. One being playbook speed, others include reliability, maintainability ect.
I think the HTTPAPI also provides reliability in the form of using multiple APIC hosts and maintainability by allowing connection variables to move to the inventory. However, yes, the suppress options are mainly for speed at the expense of some functionality. I think optimization is still okay to be used and it also gives us a document to extend if anyone finds more optimizations other than speed.

Reviewer: ok

> To find out more about a number options available to optimize the execution of playbooks, please refer to this documentation, [Optimizing Playbooks](docs/optimizing.md).

Reviewer: ```suggestion
To find out more about optimizing playbook execution, please refer to the [Optimizing Playbooks](docs/optimizing.md) documentation.
```

Author: Done
